pkgs/tools/networking/curl/default.nix

```diff
@@ -44,7 +45,10 @@
 
 assert !(gnutlsSupport && opensslSupport);
 assert !(gnutlsSupport && wolfsslSupport);
+assert !(gnutlsSupport && rustlsSupport);
 assert !(opensslSupport && wolfsslSupport);
+assert !(opensslSupport && rustlsSupport);
+assert !(rustlsSupport && wolfsslSupport);
```

Comment:
Wonder this could be done smarter with `lib.count`.

Reply:
This is great, thanks.